apptools/preferences/preferences.py

```diff
@@ -541,6 +541,14 @@ def _remove_preferences_listener(self, listener):
 
     def _set(self, key, value):
         """ Set the value of a preference in this node. """
+        
+        # everything must be unicode encoded so that ConfigObj configuration
+        # can properly serialize the data. If we receive regular str, they are
+        # supposed to be utf-8 encoded
+        if isinstance(value, str):
+            value = value.decode('utf-8')
```

Comment:
Thinking about my initial use case a bit more, what I was trying to solve was the unicode object coming from the UI (or other sources) and that had to be serialized to the preferences. I agree that the assumption of utf-8 encoded string is not good.. I'll remove that.